### PR TITLE
Minor optimizations

### PR DESCRIPTION
**Pull request by JediRhymeTrix**
In addition to this, does polarity_scores() expect a paragraph string or a list of sentences? Because if it's the latter, there is more scope for optimization as an entire for loop can be removed.

### CODE REVIEW

**Comment by JediRhymeTrix** [comment]
Working towards resolving #14

File: src/senti_classifier/senti_classifier.py [hunk 1]
```diff
@@ -175,7 +175,7 @@ def disambiguateWordSenses(sentence, word):
     bestScore = 0.0
     result = None
     for synset in wordsynsets:
-        for w in nltk.word_tokenize(sentence):
+        for w in tokens:
```
Comment (kevincobain2000) [inline src/senti_classifier/senti_classifier.py:178]: Thanks, for the optimizations.
tokens is undefined here.

Comment (kevincobain2000) [reply src/senti_classifier/senti_classifier.py:178]: https://github.com/kevincobain2000/sentiment_classifier/pull/16/commits/56b0e58a8f91a100f9bdc4674bb857016667c63a

Comment (JediRhymeTrix) [reply src/senti_classifier/senti_classifier.py:178]: should work now

Comment (JediRhymeTrix) [reply src/senti_classifier/senti_classifier.py:178]: 6f85ca5

**Pull request merged**